src/xen_helper/xen_helper.h

```diff
@@ -120,8 +124,25 @@ typedef struct xen_interface
     xentoollog_logger* xl_logger;
     xc_evtchn *evtchn;             // the Xen event channel
     int evtchn_fd;                 // its FD
+
+#ifdef ENABLE_IPT
```

Comment:
We don't need for an ifdef here either, we can have the pointer in-place even if its not used